### DISCUSSION TITLE
Web UI for plex_debrid

### DISCUSSION THREAD

**Discussion by itsToggle** [itsToggle/plex_debrid#39] [Polls]
Hey guys,

since the settings are getting more complex with every new service thats integrated, Im thinking of creating an optional Web UI for plex_debrid. A Web UI would have the following benefits / downsides:

benefits:
- easier managing of settings
- visual overview of all monitored content from all watchlists
- easier plex integration, since plex oauth could be used to register new plex users

downsides:
- python http servers can and will crash sometimes, so the overall stability would take a hit
- the programm would become more complex, and integrating new features and settings would take more time.

what are your thoughts on this?

**Comment by beljim** [comment]
Maybe have the 2 branches.
A No GUI for one fast release and a slower one for the GUI and updated less frequently, Unless it means a complete new way of coding the program.

**Reply by itsToggle** [reply]
Thats a good idea. I can make use of a web ui that I wrote for a different project, but Im not sure when I will be able to implement it. I want to do this right, so it might take a little while. Its also exam season at my uni and I cant procrestinate forever with this project.
